docs/articles/documentation/extending-testcafe/custom-reporter/reporter-methods.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: docs
+title: Reporter Methods
+permalink: /documentation/extending-testcafe/custom-reporter-plugin/reporter-methods.html
+checked: false
+---
+# Reporter Methods
+
+To create your [custom reporter](index.md#implementing-the-reporter), you need to implement the following reporter methods.
+
+> You can use [helper methods](helper-methods.md) within the reporter methods to output required data.
```

Comment:
I've written about `this` in index.md and helper-methods.md. I think, it is enough.